kmod/src/inode.c

```diff
@@ -1595,10 +1603,15 @@ int scoutfs_scan_orphans(struct super_block *sb)
 			ret = -ENOENT;
```

Comment:
This existing test for wrapping of the ino in the key is kind of nonsense, but it's made even more nonsensical by adding the add_cpu/continue above without testing it.  I think it'd be cleaner to use a for(;;scoutfs_key_inc(&key)) instead of the manual key adds so we can remove both of them and this wrapping test.

Reply:
nice!